### PR TITLE
Update runners

### PR DESCRIPTION
**Pull request by ChristianFeldmann**
Update the runners to the latest versions.
Install nasm with a using directive. Much less code.

### CODE REVIEW

File: .github/workflows/Build.yml [hunk 1]
```diff
@@ -8,28 +8,17 @@ jobs:
     strategy:
       matrix:
         include: 
-          - os: ubuntu-18.04
           - os: ubuntu-20.04
+          - os: ubuntu-22.04
           - os: macos-10.15
```
Comment (MyGodItsFull0fStars) [inline .github/workflows/Build.yml]: Shouldn't the MacOS 10.15 also be removed from this `Build.yml` since it was removed in the other commit?

**Review by MyGodItsFull0fStars — CHANGES_REQUESTED** [review]
As said in the direct comment, the MacOS OS mention likely need to be removed.
Otherwise LGTM :)

File: .github/workflows/Build.yml (hunk 1 above)
Comment (ChristianFeldmann) [reply .github/workflows/Build.yml]: I think you looked at outdated code. I removed that.

Comment (MyGodItsFull0fStars) [reply .github/workflows/Build.yml]: Yes, sorry my bad.

**Review by MyGodItsFull0fStars — APPROVED** [review]
Thank you for the PR!

**Pull request merged**